website/pages/use-cases/identity-based-access/content.json

```diff
@@ -85,7 +85,7 @@
         "content": "Create and manage policies that authorize access control throughout your infrastructure and organization."
       },
       "codeBlock": {
-        "options": { "showWindowBar": true },
+        "options": { "showChrome": true },
```

Comment:
Ran a global replace to move to `showChrome`, as `showWindowBar` will be unsupported in the next version of our `code-block` component.